YtFlowApp/Pages/ConfigListPage.xaml

```diff
@@ -46,6 +46,8 @@
                     <controls:CommandBarFlyout>
                         <controls:CommandBarFlyout.SecondaryCommands>
                             <AppBarButton Label="Shadowsocks" Click="CreateShadowsocksButton_Click"/>
+                            <AppBarButton Label="Scan QrCode(SS)" Click="QrCodeShadowsocksButton_Click"/>
```

Comment:
Use a separator to divide manual input items and scan/copy&paste stuff.

Reply:
Solved